litmus/AArch64Compile_litmus.ml

```diff
@@ -118,6 +118,16 @@ module Make(V:Constant.S)(C:Config) =
 
     let ldr_memo t = Misc.lowercase (ldr_memo t)
     let str_memo t = Misc.lowercase (str_memo t)
+    let load_literal tr_lab memo v rD lbl = match v with
+      | V32 ->
+          { empty_ins with
+            memo = sprintf "%s ^wo0, %s" memo (A.Out.dump_label (tr_lab lbl));
+            outputs=[rD]; reg_env=[(rD, word)]; }
+      | V64 ->
+          { empty_ins with
+            memo = sprintf "%s ^o0, %s" memo (A.Out.dump_label (tr_lab lbl));
+            outputs=[rD]; reg_env=[(rD, word)]; }
```

Comment:
Should not "word" be replaced by "quad"?

Reply:
good spot, will fix